server/svix-server/src/core/permissions.rs

```diff
@@ -71,6 +74,8 @@ impl FromRequestParts<AppState> for Organization {
             _ => return Err(HttpError::permission_denied(None, None).into()),
         };
 
+        tracing::Span::current().record("org_id", org_id.to_string());
```

Comment:
Instead of having this everywhere, shouldn't this just be in permissions_from_bearer so that it's centralized?